dll/win32/shell32/shell32.spec

```
@@ -456,8 +456,8 @@
 745 stub -noname Create_IUIElement
 747 stdcall SHLimitInputEdit(ptr ptr)
 748 stdcall -noname SHLimitInputCombo(ptr ptr)
-749 stub SHGetShellStyleHInstance
-750 stub SHGetAttributesFromDataObject
+749 stdcall -noname SHGetShellStyleHInstance()
```

From MSDN:
> [SHGetShellStyleHInstance is available for use in the Windows XP operating system up to and including Service Pack 2 (SP2). It is not available in later versions of Windows, such as Windows Vista.]
Deprecated. Attempts to get a handle to the Shellstyle.dll file.

(and you can verify it may as well exist on Win2003 too). So you could refine this by using:
```suggestion
749 stdcall -noname -version=0x501-0x502 SHGetShellStyleHInstance()
```

---

Fixed. Thanks @HBelusca.
Just missed that.